node/runtime/pangolin/src/lib.rs

```diff
@@ -923,7 +921,20 @@ impl dvm_rpc_runtime_api::ConvertTransaction<OpaqueExtrinsic> for TransactionCon
 }
 
 fn migrate() -> Weight {
-	<darwinia_staking::MinimumValidatorCount<Runtime>>::put(2);
+	let module = b"BSC";
+
+	migration::take_storage_value::<Vec<bsc_primitives::Address>>(
+		module,
+		b"FinalizedAuthority",
+		&[],
+	);
+	migration::take_storage_value::<bsc_primitives::BscHeader>(module, b"FinalizedCheckpoint", &[]);
+	migration::take_storage_value::<Vec<bsc_primitives::Address>>(module, b"Authorities", &[]);
```

Comment:
Can you explain more the reason about `take_storage_value`  firstly, then `remove_storage_prefix`. In my view, `remove_storage_prefix` is enough here.

Reply:
Not sure if that works with the non-map items.

Reply:
Same question, and are they the same?

And we did the same thing twice in different way here?

Reply:
If we are sure that `remove_storage_prefix` will work, then `take_storage_value` here should be unnecessary and can be removed.

We can check that afterwards using fork-off-substrate or in testnet.